client/core/core.go

```diff
@@ -1125,18 +1125,12 @@ func (c *Core) Register(form *RegisterForm) (*RegisterResult, error) {
 func (c *Core) verifyRegistrationFee(wallet *xcWallet, dc *dexConnection, coinID []byte, assetID uint32) {
 	reqConfs := dc.cfg.RegFeeConfirms
 
-	regConfs, err := wallet.Confirmations(coinID)
-	if err != nil {
-		log.Errorf("Error getting confirmations for %s: %v", hex.EncodeToString(coinID), err)
-		return
-	}
-
-	dc.setRegConfirms(regConfs)
+	dc.setRegConfirms(0)
 	c.refreshUser()
 
 	trigger := func() (bool, error) {
 		confs, err := wallet.Confirmations(coinID)
-		if err != nil {
+		if err != nil && !errors.Is(err, asset.CoinNotFoundError) {
```

Comment:
Looking at this again, may be the only real cause for concern is the initial call to `wallet.Confirmations` because the `trigger` fn would only get called when the wallet tip changes, and it's reasonable to expect that the wallet would be aware of the tx by the time it gets record of a new block.

Probably still won't hurt to play safe here, the only caveat is this trigger would be called infinitely everytime a new block is recorded _**if**_ the txid really does not exist. But then again, we did just broadcast the tx, so why would it not exist?